editor/editor_sectioned_inspector.cpp

```diff
@@ -234,7 +234,7 @@ void SectionedInspector::update_category_list() {
 			continue;
 		}
 
-		if (pi.name.find(":") != -1 || pi.name == "script" || pi.name == "resource_name" || pi.name == "resource_path" || pi.name == "resource_local_to_scene" || pi.name.begins_with("_global_script")) {
+		if (pi.name.find(":") != -1 || pi.name == "script" || pi.name == "resource_name" || pi.name == "resource_path" || pi.name == "resource_local_to_scene" || pi.name.begins_with("_global_script") || pi.name.begins_with("autoload")) {
```

Comment:
Add a comment to explain the change, as it's not obvious at a glance:

```suggestion
		// Hide the Autoload section as it has a dedicated tab in the Project Settings editor.
		if (pi.name.find(":") != -1 || pi.name == "script" || pi.name == "resource_name" || pi.name == "resource_path" || pi.name == "resource_local_to_scene" || pi.name.begins_with("_global_script") || pi.name.begins_with("autoload")) {
```